theme-data/common/common.json

```diff
@@ -14,6 +14,9 @@
           "normal": "@color-red-40",
           "hover": "@color-red-20"
         },
+        "warning": {
+          "normal": "@color-yellow-40"
+        },
         "secondary-light": "@color-white-alpha-70",
```

Comment:
this might need something added to the high contrast file

Reply:
ok, updated